pyproject.toml

```diff
@@ -705,6 +704,9 @@ voluptuous = "vol"
 
 [tool.ruff.lint.flake8-pytest-style]
 fixture-parentheses = false
+parametrize-names-type = "tuple"
+parametrize-values-row-type = "tuple"
+parametrize-values-type = "list"
```

Comment:
This is the default? Why set it?

Reply:
I had set them to test all variants, I will remove the entries.